packages/http-router/index.test-d.ts

```diff
@@ -1,6 +1,10 @@
-import { expectType } from 'tsd'
-// import middy from '@middy/core'
-import httpRouterHandler from '.'
+import middy from '@middy/core';
+import httpRouterHandler from '.';
+import { expectType } from 'tsd';
 
-const middleware = httpRouterHandler()
-expectType<any>(middleware)
+const middleware = validator({
```

Comment:
This should be `const middleware = httpRouterHandler({`

Reply:
Sorry @willfarrell. First commit to this repo, tried to do it on the fly with [github.dev](url).

Hopefully all fixed now and for future PRs I'll revert back to the IDE :)